api/src/pcapi/routes/backoffice/filters.py

```diff
@@ -204,6 +204,14 @@ def pluralize(count: int, singular: str = "", plural: str = "s") -> str:
     return plural if count > 1 else singular
 
 
+def genderize(text: str, civility: str | None) -> str:
+    if not civility:
+        return text + "(e)"
+    if civility == "Mme.":
+        return text + "e"
+    return text
```

Comment:
Bug : on n'a pas `"Mme."` dans la base.
Détail/suggestion : j'aurais plutôt mis toute autre valeur dans le cas inconnu : 
```suggestion
    if civility == GenderEnum.M.value:
        return text + "(e)"
    if civility == GenderEnum.F.value:
        return text + "e"
    return text + "(e)"
```